src/pymedphys/cli/dicom.py

```diff
@@ -37,14 +39,15 @@ def dicom_cli(subparsers):
         help='A toolbox for the manipulation of DICOM files.')
     dicom_subparsers = dicom_parser.add_subparsers(dest='dicom')
 
-    dicom_adjust_machine_name(dicom_subparsers)
-    dicom_adjust_rel_elec_density(dicom_subparsers)
-    dicom_structure_name_RED_adjust(dicom_subparsers)
+    anonymise(dicom_subparsers)
```

Comment:
Yup, this certainly be the first API seen.

Does it get crowded within the DICOM CLI docs? Should it get it's own page? What do you think?

Reply:
Hard to say. I do think the formatting of the CLI docs is pretty horrible. The heading sizes are all off. And it'd be nice to space out the sub commands and for them to have links in the side bar...

Reply:
We do have the option here to customise sphinx argparse. We can, for example, fork the project, and make the customisations we need.

Reply:
I think it's worthwhile. One day, hehe.